scripts/release.py

```diff
@@ -359,6 +361,10 @@ def prs_by_email(start_ref, end_ref):
     The dictionary is alphabetically ordered by email address
     The pull request list is ordered by merge date
     """
+    # `emails` maps from other_emails to primary email, based on people.yaml.
+    people = yaml.safe_load(requests.get(PEOPLE_YAML).text)
```

Comment:
What if `requests.get()` throws an error? We can't assume that it will always succeed.